Firestore/Source/API/FIRTransaction.mm

```diff
@@ -127,19 +130,16 @@ - (void)getDocument:(FIRDocumentReference *)document
         HARD_ASSERT(documents.size() == 1, "Mismatch in docs returned from document lookup.");
         FSTMaybeDocument *internalDoc = documents.front();
         if ([internalDoc isKindOfClass:[FSTDeletedDocument class]]) {
-          FIRDocumentSnapshot *doc = [FIRDocumentSnapshot snapshotWithFirestore:self.firestore
-                                                                    documentKey:document.key
-                                                                       document:nil
-                                                                      fromCache:NO
-                                                               hasPendingWrites:NO];
+          auto apiDoc =
+              absl::make_unique<DocumentSnapshot>(self.firestore, document.key, nil, false, false);
```

Comment:
As above, but note that this isn't test code, so the reader's context might not be the same. (In other words; still optional, but perhaps a bit less so.)

Also the nil (in addition to the bools)

There's a few more instances below too (both in this file and others).